javascriptv3/example_code/cross-services/transcription-app/src/helper.ts

```diff
@@ -26,13 +26,13 @@ window.downloadInnerHtml = function (filename, elId, mimeType) {
   link.click();
 };
 
-// Delete a row from the user interface.
+// Delete a row from the user interface
 window.deleteRow = function (rowid) {
   const row = document.getElementById(rowid);
   row.parentNode.removeChild(row);
 };
 
-// Display transcription details on user interface.
+// Display transcription details on user interface
```

Comment:
Missing period. It looks as though you removed the period here and in other places, but all sentences in comments should end with a period.